osclib/stagingapi.py

```diff
@@ -998,16 +998,17 @@ def map_ring_package_to_subject(self, project, pkg):
         """
         # it's actually a pretty stupid algorithm, but it might become more complex later
 
-        if project.endswith(':DVD'):
-            return project  # not yet
+        # assuming it is in adi staging, workaround for https://progress.opensuse.org/issues/9646
+        if self.is_adi_project(project) and self.item_exists(project, pkg):
```

Comment:
Indeed, `rm_from_prj()` is the case to which this PR relates.

The case makes sense, but the condition still seems like it should just be `if self.is_adi_project(project)` since the `:DVD` prefix is never applicable regardless of ring state or package existing in an adi staging.